src/lang/std/engineConnection.ts

```diff
@@ -1381,9 +1402,31 @@ export class EngineCommandManager extends EventTarget {
           // because in all other cases we send JSON strings. But in the case of
           // export we send a binary blob.
           // Pass this to our export function.
-          exportSave(event.data).then(() => {
-            this.pendingExport?.resolve(null)
-          }, this.pendingExport?.reject)
+          if (this.exportIntent === null) {
+            // TODO: throw a toast.
```

Comment:
just tagging where we need your help @franknoirot

Reply:
I left the console.error() call here because it's a bit more internal, but remove if needed